web_external/layout/layoutHeader.jade

```diff
@@ -82,6 +82,11 @@ nav.navbar.navbar-default.navbar-fixed-top.isic-header-wrapper
                 a(title="Upload Dataset", href="#uploadDataset")
                   i.icon-upload
                   | Upload Dataset
+            if currentUser
```

Comment:
On second thought, I think this menu item should go away entirely. Users can get to this view via either:
* a redirect after uploading
* the dataset listing page